src/shared/telemetry/util.ts

```diff
@@ -90,6 +92,32 @@ export async function getUserAgent(
     return pairs.join(' ')
 }
 
+type EnvType = 'classic' | 'codecatalyst' | ...
```

Comment:
In my comment the `...` wasn't literal, I meant to put all the other options here. Basically the comment you have on line 99 in this file is what you want to do.